ros/src/computing/planning/decision/packages/decision_maker/CMakeLists.txt

```diff
@@ -49,19 +49,16 @@ add_executable(decision_maker_node
         nodes/decision_maker/decision_maker_node_decision.cpp
         nodes/decision_maker/decision_maker_node_init.cpp
         nodes/decision_maker/decision_maker_node_callback.cpp
-        nodes/decision_maker/decision_maker_node_stateupdate.cpp
+        nodes/decision_maker/decision_maker_node_state_vehicle.cpp
+        nodes/decision_maker/decision_maker_node_state_drive.cpp
+        nodes/decision_maker/decision_maker_node_state_mission.cpp
         nodes/decision_maker/decision_maker_node_publish.cpp
         nodes/decision_maker/cross_road_area.cpp
         )
 target_link_libraries(decision_maker_node ${catkin_LIBRARIES})
 add_dependencies(decision_maker_node ${catkin_EXPORTED_TARGETS})
 
-# This function will be integrated into decision_maker soon.
-add_executable(planner_selector_node nodes/planner_selector/planner_selector_node.cpp)
```

Comment:
"planner_selector_node.cpp" and "planner_selector.hpp" are excluded from CMakeLists, but the files remain.
Also, "planner_selector.hpp" conflicts.
Should not they be deleted?

Reply:
@yk-fujii 
Is it ok to delete it?

Reply:
I removed planner_selector and parameters on other related nodes.